### DISCUSSION TITLE
Welcome to cookieclicker Discussions!

### DISCUSSION THREAD

**Discussion by Chickenroast** [Chickenroast/cookieclicker#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by googoogagga** [comment]
here's a save: Mi4wNTJ8fDE3MzQ2NTkzOTcyNDk7MTczNDY1OTM5NzMwMzsxNzM0NjU5NjI5MDk2O1BybztpamxreDswLDEsMCwwLDAsMCwwfDExMTExMTAxMTAwMTAwMTAwMTAxMDExMDAwMXxOYU47TmFOOzEyOTswO0luZmluaXR5OzA7OTsxNztJbmZpbml0eTsxOzA7MDswOy0xOzA7MDtJbmZpbml0eTswOzE0OzA7MjU4NzU1MTsxO2Zvb2xzO0luZmluaXR5OzEwO0luZmluaXR5O0luZmluaXR5OzA7MDswOy0xOy0xOy0xOy0xOy0xOzI3OzA7MDswOzUwOzA7MDsyMDk7MjEwOzE3MzQ2NTkzOTczMDM7MDswOzs0MTswOzA7SW5maW5pdHk7NTA7MDswO3wzMjAxLDMyMDEsSW5maW5pdHksMTAsLDAsMzIwMTsxOTAwLDE5MDAsTmFOLDEwLCwwLDE5MDA7MTkwMCwxOTAwLE5hTiwxMCwxNzM0NjU5NjI5ODE1OjA6MTczNDY1OTM5NzI3NTowOjA6MDowOjA6MTczNDY1OTM5NzI3NTogMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMSAwOjA6MTQ6MzA6MDowOjE0OjA6MTQ6NzQ6MTQ6OTY6MDowOjE0OjEyOjE0OjU1OjA6MDowOjA6MDowOjA6MDowOjA6MDowOjA6MDoxNDo2NjoxNDo3ODoxNDo4NjowOjA6MTQ6ODQ6MDowOjA6MDowOjA6MDowOjA6MDoxNDo3NzowOjA6MDowOjA6MDowOjA6MDowOjE0OjU0OjE0OjcxOjE0OjA6MDowOiwwLDE5MDA7MTQwMCwxNDAwLE5hTiwxMCwsMCwxNDAwOzE0MDAsMTQwMCxOYU4sMTAsLDAsMTQwMDsxNDAwLDE0MDAsTmFOLDEwLDA6MDoxOjA6MDogNDc5NDozOjMyODo0MjM6MDowOjA6MCEyNTcyOjI6NzU6MzI6MDowOjA6MCExMzc4OjQ6LTEyOjQ5NDowOjA6MDowITcyNDY6Mjo0MDM6NTEwOjA6MDowOjAhNDc3NDo0OjQzOjIyODowOjA6MDowITk2NTU6Mjo0MjM6NDU5OjA6MDowOjAhOTkxMjoyOi00MjoxMzY6MDowOjA6MCExMzQ4MTo1OjIyNTo1NjY6MDowOjA6MCExMDM3ODo1OjI3MjoyNTI6MDowOjA6MCExMDg2MzozOjM3OjUxNDowOjA6MDowITExOTY2OjU6LTE2OjUzNzowOjA6MDowITE1MDE3OjM6MTA5OjIwOTowOjA6MDowITEyODI5OjA6LTQ4OjEyNDowOjA6MDowITEzODU1OjU6MTA6NTowOjA6MDowITE1MDQzOjE6NzI6MjQxOjA6MDowOjAhMTYwMzk6NTozMDo1ODg6MDowOjA6MCExODE3NDo1OjE4OjU2MjowOjA6MDowITE5NDUzOjI6LTE2Nzo3MzowOjA6MDowISAwLDAsMTQwMDsxNDAwLDE0MDAsTmFOLDEwLC0xLy0xLy0xIDMgMTczNDY1OTM5NzI5OSAwLDAsMTQwMDsxNDAwLDE0MDAsTmFOLDEwLDcuMjA3NDAzODI1ODIzMTE4IDAgMCAwLDAsMTQwMDsxNDAwLDE0MDAsTmFOLDEwLCwwLDE0MDA7MTQwMCwxNDAwLE5hTiwxMCwsMCwxNDAwOzE0MDAsMTQwMCxOYU4sMTAsLDAsMTQwMDsxNDAwLDE0MDAsTmFOLDEwLCwwLDE0MDA7MTQwMCwxNDAwLE5hTiwxMCwsMCwxNDAwOzE0MDAsMTQwMCxOYU4sMTAsLDAsMTQwMDsxNDAwLDE0MDAsTmFOLDEwLCwwLDE0MDA7MTQwMCwxNDAwLE5hTiwxMCwsMCwxNDAwOzE0MDAsMTQwMCxOYU4sMTAsLDAsMTQwMDsxNDAwLDE0MDAsTmFOLDEwLCwwLDE0MDA7MTQwMCwxNDAwLE5hTiwxMCwsMCwxNDAwOzE0MDAsMTQwMCxOYU4sMTAsLDAsMTQwMDt8MTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMTExMTExMTExMTExMTExMTExMDAwMDExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMTAxMDExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMDEwMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMTAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMDExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAwMDAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMXwxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMDAwMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExfDE5LDEwODAwMCwxMDM1ODgsMzt8
